ros-rviz-image.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-image/iron-image.html">
+<link rel="import" href="../paper-checkbox/paper-checkbox.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="imports.html">
+
+<dom-module id="ros-rviz-image">
+  <template>
+    <paper-input label="Image topic" value="{{topic}}"></paper-input>
+    <paper-checkbox checked="{{continuous}}">Continuously update</paper-checkbox>
+    <paper-input label="Transport hints" value="{{transport_hints}}"></paper-input>
+    <iron-image id="xy_image" src="[[image_source]]" alt="iron-image" ></iron-image>
```

Comment:
Maybe the alt text could be something like "no image". If we want to be fancy, we could also add a `title` with the image topic.